Jenkinsfile

```diff
@@ -15,12 +17,15 @@ node('windows') {
 
```

Comment:
Sercomm installation should be removed after merge PR https://github.com/ingeniamc/ingenialink/pull/83